.github/ISSUE_TEMPLATE/---mrtk-task.md

```diff
@@ -2,16 +2,27 @@
 name: "\U0001F4DD Mixed Reality Toolkit Task"
 about: File an outstanding task for MRTK vNext.
 ---
+> Necessary checks for creating a new vNext Task:
+> * Prefix the Title of the Task with "vNext Task: "
```

Comment:
Why when we can just add the label?

Reply:
hahahaha, I tested.

WE can add labels, normal users cannot.
And as with the other vNext tasks, we have pre-fixed them for now to make them clear vNext Tasks and not issues or suggestions.

Reply:
Yeah, that's what we're saying. We'll be the ones adding the labels, since only users with write permissions can. Whether it's prefixed or not, we'll be adding the labels, so the prefix seems redundant.

Reply:
It's been the standard for all the vNext tasks so far to make them stand out. :D

Reply:
Right, and this is a request to change the way things are done going forward. We don't have to do the same thing forever!